experimental/generation/runbot/RunBot.cs

```diff
@@ -19,34 +19,37 @@
 using Microsoft.Bot.Builder.Dialogs.Choices;
 using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
 using Microsoft.Bot.Builder.Skills;
+using Microsoft.Extensions.Configuration;
 
 namespace RunBotServer
 {
     public class RunBot : ActivityHandler
     {
-        private IStatePropertyAccessor<DialogState> dialogStateAccessor;
-        private DialogManager dialogManager;
-        private readonly ResourceExplorer resourceExplorer;
+        private IStatePropertyAccessor<DialogState> _dialogStateAccessor;
```

Comment:
I'm a big fan of the underscore prefix as a style as it reduces the chance of bugs :)

Reply:
Me too.  I inherited this code.  You want me to change it all?